l3build.lua

```diff
@@ -173,21 +325,22 @@ if options["target"] == "check" then
     end
   end
 end
+local config_1 = checkconfigs[1]
 if #checkconfigs == 1 and
-   checkconfigs[1] ~= "build" and
+   config_1 ~= "build" and
    (options["target"] == "check" or options["target"] == "save" or options["target"] == "clean") then
-   local config = "./" .. gsub(checkconfigs[1],".lua$","") .. ".lua"
+   local config = work_dir .. gsub(config_1, ".lua$","") .. ".lua"
```

Comment:
We look for config files in the same directory where `build.lua` lives, id est the `work_dir`.